persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/StargateQueryHandler.java

```diff
@@ -125,7 +127,15 @@ public void register(QueryInterceptor interceptor) {
   @Override
   public Single<ResultMessage.Prepared> prepare(
       String query, QueryState queryState, Map<String, ByteBuffer> customPayload) {
-    return QueryProcessor.instance.prepare(query, queryState, customPayload);
+    return QueryProcessor.instance
+        .prepare(query, queryState, customPayload)
+        .map(
```

Comment:
Why is this different from the other two?

Reply:
because it return `Single<ResultMessage.Prepared>` and not `ResultMessage.Prepared`

Reply:
Oh, right.